apps/studio/src/migration/20251116_tab_history_updates.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  name: '20251116-tab-history-updates',
+  async run(runner) {
+    const queries = [
+      `ALTER TABLE tabs ADD COLUMN lastActive datetime`,
```

Comment:
We'll need to add a reverse index to `lastActive`, seeing as you're using it in a DESC search.

Reply:
Probably also need an index on the deletedAt column

Reply:
sure thing